packages/Radio/src/Radio.js

```diff
@@ -98,6 +102,7 @@ function Radio(props) {
     ref: inputRef,
     tabIndex,
     type: "radio",
+    value,
```

Comment:
In results questionnaires the value is scraped from the input as part of the form submission. Setting it on the input here allows us to retain this approach until the business logic for questionnaires submission is updated and changed.

Reply:
oh interesting, why not just pass it through `moreProps` like
```
const inputProps = {
    "aria-describedby": ariaDescribedBy,
    readOnly: true,
    onClick,
    checked: isChecked,
    disabled: isDisabled,
    id: radioId,
    name,
    onKeyDown: handleKeyDown,
    onKeyUp: handleKeyUp,
    ref: inputRef,
    tabIndex,
    type: "radio",
   ...moreProps,
}
```

Reply:
because `moreProps` is currently being spread to the container div on line 110, not the input so I don't think we want it on both.